anime_downloader/commands/dl.py

```diff
@@ -130,7 +130,7 @@ def command(ctx, anime_url, episode_range, url, player, skip_download, quality,
             util.print_episodeurl(episode)
 
         if player:
-            util.play_episode(episode, player=player, title=f'{anime.title} - Episode {episode.ep_no}')
```

Comment:
Add something like `episode_ramge = f"0:{len(animes)}" if not episode_ramge else episode_range` before this line

Reply:
Had to modify util.py to get this to fully work